content/en/docs/architecture/step-registry.md

```diff
@@ -175,6 +175,28 @@ ref:
 By default, only secrets in the `test-credentials` namespace will be available for mounting into test steps. Please follow the secret-mirroring
 [documentation](https://github.com/openshift/release/blob/master/core-services/secret-mirroring/README.md) to set up a custom secret in that namespace**
 
+##### Custom Bitwarden items
+
+Adding a new Bitwarden item to create a new secret or inject it in an existing one, requires few minor steps.
+First, the user has to create the Bitwarden item and share it with the *OpenShift TestPlatform (CI)* collection,
```

Comment:
Are we certain that users can get new accounts made for them in BitWarden? Is there no limit? This doc should tell them to request those from DPP as well.

Reply:
I will sync today with James for that as well.